scripts/s3/s3api/multipart_operations.py

```diff
@@ -75,30 +75,30 @@ def __init__(self, access_key: str, secret_key: str, endpoint_url: str, use_ssl:
 
     async def execute_multipart_workload(self):
         """Execute multipart workload for specific duration."""
-        mpart_bucket = "s3mpart-bkt-{}-{}".format(self.test_id, perf_counter_ns())
+        mpart_bucket = f"s3mpart-bkt-{self.test_id}-{perf_counter_ns()}"
         await self.create_bucket(mpart_bucket)
         while True:
-            LOGGER.info("Iteration %s is started for %s...", self.iteration, self.test_id)
+            self.log.info("Iteration %s is started for %s...", self.iteration, self.session_id)
             try:
-                LOGGER.info("Bucket name: %s", mpart_bucket)
-                s3mpart_object = "s3mpart-obj-{}-{}".format(self.test_id, perf_counter_ns())
-                s3_object = "s3-obj-{}-{}".format(self.test_id, perf_counter_ns())
-                LOGGER.info("Object name: %s", s3mpart_object)
+                self.log.info("Bucket name: %s", mpart_bucket)
+                s3mpart_object = f"s3mpart-obj-{self.test_id}-{perf_counter_ns()}"
+                s3_object = f"s3-obj-{self.test_id}-{perf_counter_ns()}"
+                self.log.info("Object name: %s", s3mpart_object)
                 if isinstance(self.object_size, dict):
                     file_size = random.randrange(self.object_size["start"], self.object_size["end"])
                 else:
                     file_size = self.object_size
-                LOGGER.info("File size: %s GiB", (file_size / (1024 ** 3)))
+                self.log.info("File size: %s GiB", (file_size / (1024 ** 3)))
                 number_of_parts = random.randrange(self.part_range["start"], self.part_range["end"])
-                LOGGER.info("Number of parts: %s", number_of_parts)
+                self.log.info("Number of parts: %s", number_of_parts)
                 assert number_of_parts <= 10000, "Number of parts should be equal/less than 10k"
                 single_part_size = round(file_size / number_of_parts)
-                LOGGER.info("single part size: %s MiB", single_part_size / (1024 ** 2))
+                self.log.info("single part size: %s MiB", single_part_size / (1024 ** 2))
                 response = await self.create_multipart_upload(mpart_bucket, s3mpart_object)
                 assert response["UploadId"], f"Failed to initiate multipart upload: {response}"
```

Comment:
Redundant assert. if initiate multipart fails, then this assert won't get executed, the exception will be cached by except block.

Reply:
This is there in multiple instances below. Please use asserts for validating inputs and not for validating s3 operation failures.